tests/h/conftest.py

```diff
@@ -117,7 +117,7 @@ def restart_savepoint(session, transaction):
 @pytest.fixture(scope='session', autouse=True)
 def deform():
     """Allow tests that use deform to find our custom templates."""
-    form.init()
+    form.includeme(config=mock.Mock())
```

Comment:
Hmm. I guess this doesn't cause tests to fail, but it certainly seems a little odd, given the docstring.